### PR TITLE
GitHub action

### PR DESCRIPTION
**Pull request by robkooper**
build node and docker image.

### CODE REVIEW

File: .github/workflows/ci.yaml [hunk 1]
```diff
@@ -0,0 +1,122 @@
...
+      if: github.repository == env.MAIN_REPO
+      uses: elgohr/Publish-Docker-Github-Action@2.22
+      env:
+        BRANCH: ${{ env.GITHUB_BRANCH }}
+        VERSION: ${{ env.VERSION }}
+        BUILDNUMBER: ${{ github.run_number }}
+        GITSHA1: ${{ github.sha  }}
+      with:
+        registry: hub.ncsa.illinois.edu
+        name: ${{ github.repository_owner }}/${{ matrix.IMAGE }}
```
Comment (longshuicy) [inline .github/workflows/ci.yaml:118]: i think the name of the repository should be `incore/frontend` @robkooper

Comment (ywkim312) [reply .github/workflows/ci.yaml]: Github action should work now. Thank you very much @robkooper!

**Comment by robkooper** [comment]
When merging, please squash and merge.

**Comment by longshuicy** [comment]
closing #20

**Pull request merged**